xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextExtension.xtend

```diff
@@ -169,20 +173,20 @@ class GeneratorConfig {
 
 @Accessors
 class GeneratedAnnotationOptions {
-	boolean active
-	boolean includeDate
-	String comment
+	@Input boolean active
```

Comment:
`@Optional` does not seem to be needed

Reply:
A primitive can never be null, so Optional wouldn't make sense.